renku_notebooks/.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
+    "configurations": [
+    {
+      "name": "Python: Attach",
+      "type": "python",
+      "request": "attach",
+      "connect": {
+        "host": "localhost",
+        "port": 5678
+      }
+    }
+    ]
+}
```

Comment:
@olevski you can configure your vscode to add a newline at the end of each file automatically.